biom/table.py

```diff
@@ -2700,7 +2700,7 @@ def subsample(self, n, axis='sample', by_id=False, with_replacement=False):
         with_replacement : boolean, optional
             If `False` (default), subsample without replacement. If `True`,
             resample with replacement via the multinomial distribution.
-            Ignored when `by_id` is `True`.
+            If `True` and `by_id=True`, a ValueError is raised.
```

Comment:
Sorry to be a stickler, but can you use the numpydoc structure? See `table.max` for an example of describing the exceptions